bots/pr/src/main/java/org/openjdk/skara/bots/pr/ApprovalCommand.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.skara.bots.pr;
+
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.util.List;
+import org.openjdk.skara.forge.PullRequest;
+import org.openjdk.skara.issuetracker.Comment;
+import org.openjdk.skara.issuetracker.Issue;
+
+public class ApprovalCommand implements CommandHandler {
+    // The tags to re-run the CheckWorkItem of the PRBot.
+    private static final String APPROVAL_MARKER = "<!-- approval: 'yes' -->";
+    private static final String DISAPPROVAL_MARKER = "<!-- approval: 'no' -->";
+
+    private static void showHelp(PrintWriter writer) {
+        writer.println("""
+                usage: `/approval [yes|no|y|n]`
```

Comment:
Perhaps a minor issue, but I think the command should be `/approve` instead of `/approval`. We have verbs in the other commands, such as `/sponsor` and `/integrate`.

Reply:
I agree the command name `approve` is better. Do the related class names and method names need to be adjusted? Or only need to revise the command name and its help info?

```
            Map.entry("approval", new ApprovalCommand())  --> change to `approve`
```

Reply:
I changed the command name `approval` to `approve` just now.

Reply:
I think the name of the command class should match the command, so please change to `ApproveCommand`

Reply:
This name has been changed. So as its test class. Please see the newest code.
If there are other names that need to be adjusted, please help to point out.